site/docs/reference/connectors-policymanager/README.md

```diff
@@ -0,0 +1,29 @@
+# Policy Manager Service
+
```

Comment:
I suggest adding a description of what the policy manager service does - i.e. gets data governance decisions for the data sets indicated in FybrikApplication yaml based on the context indicated

Reply:
@rohithdv , there was a race between the PR merging and @simanadler's comment above. Could you please add the required description in a separate PR.

Reply:
sure @roytman.